### PR TITLE
Combat tag

### PR DESCRIPTION
**Pull request by SciathGH**

### CODE REVIEW

File: src/main/kotlin/net/starlegacy/feature/starship/DeactivatedPlayerStarships.kt [hunk 1]
```diff
@@ -3,19 +3,25 @@ package net.starlegacy.feature.starship
 import it.unimi.dsi.fastutil.longs.LongOpenHashSet
 import it.unimi.dsi.fastutil.objects.Object2ObjectOpenHashMap
 import java.io.File
-import java.util.UUID
+import java.util.*
```
Comment (Peter-Crawley) [inline src/main/kotlin/net/starlegacy/feature/starship/DeactivatedPlayerStarships.kt:6]: No wildcard imports.

**Review by Peter-Crawley — CHANGES_REQUESTED** [review]
Actions check failed.

File: src/main/kotlin/net/starlegacy/feature/starship/DeactivatedPlayerStarships.kt (hunk 1 above)
Comment (SciathGH) [reply src/main/kotlin/net/starlegacy/feature/starship/DeactivatedPlayerStarships.kt:6]: I blame intellij

File: src/main/kotlin/net/starlegacy/feature/starship/active/ActiveStarship.kt [hunk 2]
```diff
@@ -3,6 +3,7 @@ package net.starlegacy.feature.starship.active
 import com.destroystokyo.paper.Title
 import com.google.common.collect.HashBiMap
 import com.google.common.collect.HashMultimap
+import com.sun.jna.platform.unix.X11.Atom
```
Comment (Peter-Crawley) [inline src/main/kotlin/net/starlegacy/feature/starship/active/ActiveStarship.kt:6]: ?????

**Review by Peter-Crawley — CHANGES_REQUESTED** [review]

File: src/main/kotlin/net/starlegacy/feature/starship/active/ActiveStarship.kt (hunk 2 above)
Comment (SciathGH) [reply src/main/kotlin/net/starlegacy/feature/starship/active/ActiveStarship.kt:6]: heh?

**Comment by SciathGH** [comment]
never happened

**Comment by Peter-Crawley** [comment]
Janky, but i'll tollerate it for IonCore

**Pull request merged**